package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "business-edit-ui",
-  "version": "4.7.1",
+  "version": "4.7.2",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "business-edit-ui",
-      "version": "4.7.0",
```

Comment:
@PaulGarewal please use `npm i` to update this file, if you manually edit it, you can miss places to update the version